SpywareFilter/sections/specific.txt

```diff
@@ -79,6 +79,8 @@ semprotyuk.com##div[style^="margin-bottom:150px;"][style*="padding-right:2em;"]
 ||n11scdn.akamaized.net/assets/javascripts/ha.js
 ||playbrain.io/analytics/
 ||linkwithin.com/pixel.png
+||hotstar.com^/*track
```

Comment:
No, I mean like that `||hotstar.com/*/track`

Reply:
Oh yeah.